### DISCUSSION TITLE
help: Grid Template Rows not equally distributing width in grid-rows-{n}

### DISCUSSION THREAD

**Discussion by ElliottCuvelier** [tailwindlabs/tailwindcss#7880] [Help] [answered]
**What version of Tailwind CSS are you using?**

2.2.4 on the app where I encountered the issue but also tried it in the playground (3.0.23) and got the same behavior.

**What build tool (or framework if it abstracts the build tool) are you using?**

postcss: 8, Next.js 12.1.0

**What version of Node.js are you using?**

v16.13.0

**What browser are you using?**

Firefox 98.0.1

**What operating system are you using?**

macOS 12.2.1

**Reproduction URL**

https://play.tailwindcss.com/PqkCk2fzJD

**Describe your issue**

The width of the columns of a grid element are not equally distributed when the content of 1 child element of a grid element is bigger than the other child's content:

![Screen Shot 2022-03-17 at 10 59 58 PM](https://user-images.githubusercontent.com/9059917/158901653-00730443-9daf-463d-90df-66352fc88d51.png)

![Screen Shot 2022-03-17 at 10 59 31 PM](https://user-images.githubusercontent.com/9059917/158901603-2cf8d7fd-54b4-4b42-9a28-db9e38803d95.png)

Am I doing something wrong or is there some kind of an issue or is this a "normal" behavior for a grid element?

Thanks!

**Comment by adamwathan** [comment, marked as answer]
Hey! This is normal CSS grid behavior — the `grid-auto-columns` property is set to `auto` by default. If you want automatically inserted columns to all be the same width, you want to set it to `minmax(0, 1fr)`, which you can do in Tailwind using the `auto-cols-fr` class:

https://play.tailwindcss.com/RhFSXQpz7I

**Reply by ElliottCuvelier** [reply]
That simple... 😅 thanks!
